charts/oes/values.yaml

```diff
@@ -392,10 +429,12 @@ visibility:
     ##
 
     ## Autopilot integration options
-    configuredConnectors: JIRA,GIT,AUTOPILOT,SONARQUBE,JENKINS
+    configuredConnectors: JIRA,GIT,AUTOPILOT,SONARQUBE,JENKINS,AQUAWAVE
 
     ## Logger level
     logLevel: ERROR
+    gate:
+      url: http://OES_GATE_IP:8084
```

Comment:
Let's define OES-UI url and OES-Gate url at a single place and use them all over. Not define it under every section

Reply:
Done